vendor/github.com/moby/swarmkit/v2/api/types.proto

```diff
@@ -90,6 +90,8 @@ message ResourceRequirements {
 	// to the container OS's default - generally 60, or the value predefined in
 	// the image; set to -1 to unset a previously set value
 	google.protobuf.Int64Value memory_swappiness = 4;
+
```

Comment:
Wondering where the empty lines here originated from, or is this some change in the code generating these? (just curious where the changes came from)